firmware/application/apps/ui_scanner.cpp

```diff
@@ -490,7 +491,7 @@ ScannerView::ScannerView(
     // Button to add current frequency (found during Search) to the Scan Frequency List
     button_add.on_select = [this](Button&) {
         FreqmanDB db;
-        if (db.open(get_freqman_path(freqman_file), /*create*/ true)) {
+        if (db.open(get_freqman_path(freqman_file, dir_profile::ProfileUser), /*create*/ true)) {
```

Comment:
In current firmware, if someone opens a FREQMAN file and presses "ADD", the frequency will be added to that FREQMAN file.  With the proposed folder structure, it looks like a new /USR/FREQMAN/file will be created that only contains the added frequencies.  The next time the user runs the Scanner app, they'll have to choose between loading the USR file (which will only contain the few frequencies they added) or the system file (which whil not contain any of the frequencies they added).  Nobody ever liked that the Scanner app modified the original FREQMAN files anyway, but with the proposed folder structure I think we should provide a simple way for the user to copy all of the frequencies from the system file to the USR file, and perhaps complain if an attempt is made to edit the system file, as you do in the FreqMan app.

Reply:
fixed